cfg/config_util_test.go

```diff
@@ -23,3 +23,32 @@ import (
 func Test_DefaultMaxParallelDownloads(t *testing.T) {
 	assert.GreaterOrEqual(t, DefaultMaxParallelDownloads(), 16)
 }
+
+func TestIsFileCacheEnabled(t *testing.T) {
```

Comment:
Please convert it into a table test.

Reply:
Did not do it initially as the test was moved from a different file. Done now :)